src/app/modules/design-patterns/services/roulette.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import * as faker from 'faker';
+import { Subject } from 'rxjs';
+
+import { getRandomRouletteColor } from 'utils/helpers';
+import { RoulettetBetModel } from '../observer/models/roulette-bet.model';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RouletteService {
+  bets = new Subject<RoulettetBetModel>();
+
+  generateRandomBets(): RoulettetBetModel[] {
+    const bets = [];
```

Comment:
add blank line below

Reply:
why? all of them are variables and are assigning values. I mean, all assignations are grouped

Reply:
multiline normally should be separated :P